test/test-applications/integrations/TestApplication.GraphQL/Properties/launchSettings.json

```diff
@@ -16,14 +16,21 @@
 
         "CORECLR_ENABLE_PROFILING": "1",
         "CORECLR_PROFILER": "{918728DD-259F-4A6A-AC2B-B85E1B658318}",
-        "CORECLR_PROFILER_PATH": "$(SolutionDir)bin\\tracer-home\\win-x64\\OpenTelemetry.AutoInstrumentation.Native.dll",
-        "DOTNET_STARTUP_HOOKS": "$(SolutionDir)bin\\tracer-home\\netcoreapp3.1\\OpenTelemetry.AutoInstrumentation.StartupHook.dll",
+        "CORECLR_PROFILER_PATH_64": "F:\\Dev\\forks\\rassk\\opentelemetry-dotnet-instrumentation\\bin\\tracer-home\\win-x64\\OpenTelemetry.AutoInstrumentation.Native.dll",
```

Comment:
I don't think that you meant to commit most of the changes in in this file.

Reply:
Good catch!